pkg/beyla/config.go

```diff
@@ -242,7 +242,7 @@ func (c *Config) Validate() error {
 		return ConfigError(fmt.Sprintf("error in exclude_services YAML property: %s", err.Error()))
 	}
 	if !c.Enabled(FeatureNetO11y) && !c.Enabled(FeatureAppO11y) {
-		return ConfigError("missing at least one of BEYLA_NETWORK_METRICS, BEYLA_EXECUTABLE_NAME or BEYLA_OPEN_PORT property")
+		return ConfigError("missing to enable application discovery or network metrics. Check documentation")
```

Comment:
All the previously mentioned options were deprecated and removed from the documentation, so I think it was better to remove them from the error message.